nix/docker/k8s/cardano-nodes.yaml

```diff
@@ -0,0 +1,190 @@
+#
+# NAMESPACE=cardano
+# kubectl create namespace $NAMESPACE
+#
+
+# GCE Storage Class
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: cardano-standard-rwo
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
+  fstype: ext4
+  replication-type: none
+
+# Relay ###############################################################################################################
+
+# Relay Volume Claim
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: relay-data
+  labels:
+    app: cardano
+    pvc: relay-data
+spec:
+  storageClassName: cardano-standard-rwo
+  accessModes: [ ReadWriteOnce ]
+  resources:
+    requests:
+      storage: 10G
```

Comment:
Curious how `10G` was chosen - should this align to the "Stake Pool minimum system requirements" of `30G` -- https://iohk.zendesk.com/hc/en-us/articles/900001208966-Stake-Pool-Minimum-System-Requirements?

Reply:
The Cardano block store currently needs ...

```
 $ du -h /mnt/disks/data00/
392M    /mnt/disks/data00/ledger
6.2G    /mnt/disks/data00/immutable
6.6G    /mnt/disks/data00/
```

It is set to 10G to safe cost for these initial test clusters. It will probably be enough for quite a bit longer

Reply:
In the cardano-node release notes 20G is listed as the “required” free disk space for a pool. Those requirements are usually over-zealous, but disk is cheap and having some headroom would be preferable to running out of disk space soon 🙂